packages/@react-aria/menu/docs/useMenuTrigger.mdx

```diff
@@ -120,7 +120,8 @@ function MenuButton(props) {
       {state.isOpen &&
         <MenuPopup
           {...props}
-          domProps={menuProps}
+          {...menuProps}
+          aria-label={props.label}
```

Comment:
I believe we can get rid of this `aria-label` now that we are propagating the `aria-labelledby` from useMenuTrigger
```suggestion
```